src/pages/projects.js

```diff
@@ -266,7 +266,7 @@ const ProjectDescriptions = () => (
               <b>Patches</b>
             </a>{" "}
             |{" "}
-            <a href="https://travis-ci.com/github/kernel-patches/bpf/pull_requests">
+            <a href="https://github.com/kernel-patches/bpf/pulls">
```

Comment:
Maybe `https://github.com/kernel-patches/bpf/actions` to point to the GitHub CI jobs?

Reply:
I personally like the PR site a bit better since it provides a nicer overview of which patches it picked up from patchwork and I can directly jump to these, see conflicts etc as well as have links to the CI. At least from my PoV, slightly more pref towards the pulls.